resources/text/commands/fight.json

```diff
@@ -44,6 +44,10 @@
 					"direct": "Vous devez atteindre le **LVL {level}** avant de combattre !",
 					"indirect": "Cet adversaire n'a pas encore atteint le **LVL {level}** et ne peut donc pas combattre !"
 				},
+				"dead": {
+					"direct": "{pseudo}, Il n'est pas possible de combattre en étant K.O! Vous pouvez revivre en utilisant la commande {command:respawn} pour pouvoir continuer à jouer.",
+					"indirect": "Cette personne ne semble pas en état de combattre pour le moment !"
```

Comment:
Pourquoi dupliquer le message ? Et tu l'as pas mis en anglais

Reply:
parce que le direct est pas le meme

Reply:
je vais fix pr l'anglais

Reply:
je réitère ma demande aussi, pourquoi dupliquer le message

Reply:
Non mais c'est le même message que cantFightStatus.indirect